cyphejor.el

```diff
@@ -148,9 +156,15 @@ information."
   (funcall (if cyphejor-mode #'add-hook #'remove-hook)
            'after-change-major-mode-hook
            #'cyphejor--hook)
+  (funcall (if cyphejor-mode #'add-hook #'remove-hook)
+           'buffer-list-update-hook
+           #'cyphejor--fundamental-mode-hook)
   (if cyphejor-mode
       (advice-add 'wdired-change-to-dired-mode :after #'cyphejor--hook)
-    (advice-remove 'wdired-change-to-dired-mode #'cyphejor--hook)))
+    (advice-remove 'wdired-change-to-dired-mode #'cyphejor--hook))
+  (when cyphejor-mode
```

Comment:
This may be a problematic feature. What happens when I decide to disable `cyphejor-mode`? The modes affected by this loop will continue to live with changed names (at least from the code alone it seems like that would be the case, I haven't tried).